### PR TITLE
Adding Andre to the website

### PR DESCRIPTION
**Pull request by ghost**
Added bio and picture.

### CODE REVIEW

**Comment by richardjwild** [comment]
Can you amend the bio please to:

```
Andre is a software developer from Brazil who discovered his passion for programming while working as tech support for an ERP in the fashion industry, later he become a developer in the same company working with Delphi.

A year and half later he joined another company as the only developer in the company to build an event management system using Delphi.

In 2017 he decided to move to Europe and landed a job in the gaming industry in London, where he stayed working in the payments team for almost a year, until he joined Codurance.
```

**Comment by ghost** [comment]
Hi @richardjwild, I've amended the bio as asked. 

Thanks,
Andre.

File: _data/team.yml [hunk 1]
```diff
@@ -583,6 +583,16 @@
   twitter: SIR_XOPHER
   github: SirXOPHER
 
+- id: atorres
+  name: André Guelfi Torres
+  image: /assets/custom/img/team/andre_torres.jpg
+  role: Apprentice
+  fullDescription:
+    <p>Andre is a software developer from Brazil who discovered his passion for programming while working as tech support for an ERP in the fashion industry, later he become a developer in the same company working with Delphi.</p>
```
Comment (MiyamotoAkira) [inline _data/team.yml]: "fashion industry. Later, he become"

File: _data/team.yml [hunk 2]
```diff
@@ -583,6 +583,16 @@
...
+    <p>A year and half later he joined another company as the only developer in the company to build an event management system using Delphi.</p>
```
Comment (MiyamotoAkira) [inline _data/team.yml]: "another company, as the only developer, to build"

**Review by MiyamotoAkira — CHANGES_REQUESTED** [review]

**Comment by richardjwild** [comment]
I am happy with this PR, @MiyamotoAkira it is up to you now.

**Comment by ghost** [comment]
Hi @MiyamotoAkira. 

Changes done, 

Thanks. 
Andre.

**Review by MiyamotoAkira — APPROVED** [review]

**Pull request merged**